tests-system/lobster-cpptest/rbt-valid-source-input-file-paths/one-valid-file-scenario/expected-output/stdout.txt

```diff
@@ -0,0 +1,587 @@
+Written 5 lobster items to "cpptest.lobster".
+{
```

Comment:
Does `lobster-cpptest` really print the output to `stdout`, too? The config file of this test case specifies an output file.

Reply:
Yes. If the config file does not cover all the markers provided in cpp files, the lobster-cpptest tool prints the rest of the markers wo references to stdout.

Relevant link https://github.com/bmw-software-engineering/lobster/blob/969f68a53c928850a0e393da355aa2ada187854b/documentation/manual-lobster_cpptest.md?plain=1#L94